pjsip-apps/src/pjsua/pjsua_app_config.c

```diff
@@ -95,6 +95,7 @@ static void usage(void)
 #endif
     puts  ("  --set-qos           Enable QoS tagging for SIP and media.");
     puts  ("  --no-mci            Disable message composition indication (RFC 3994)");
+    puts  ("  --set-loam          Enable Late offer answer model");
```

Comment:
When implemented as pjsua param, all outgoing calls will not have an SDP offer. What if this is implemented as pjsua command (menu), so user has alternatives without restarting the pjsua app?

Reply:
Yes, sure. I will correct it.